packages/elements-core/src/styles.css

```diff
@@ -69,20 +69,13 @@
 
 /* HttpOperation */
 
-.sl-elements .HttpOperation .JsonSchemaViewer .MarkdownViewer p {
+.sl-elements .HttpOperation .JsonSchemaViewer .sl-markdown-viewer p, .sl-elements .HttpOperation__Parameters .sl-markdown-viewer p, .sl-elements .Model .JsonSchemaViewer .sl-markdown-viewer p {
```

Comment:
This is just fixing the 3 rules on the left (they had the old mdv classname), the real "fix" is the `padding-right`

Reply:
Consolidating 3 rules into 1 instead of simply fixing the classnames is the kind of unhelpful refactoring that means I spend 20 minutes resolving a merge conflict 5 days later instead of being able to Just Merge It.